api/AudioParam.json

```
@@ -517,15 +517,19 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "14"
+              "version_added": "14",
+              "notes": "Before version 64, the gain value of a `GainNode` would perform a smooth interpolation to prevent dezippering (instead of changing instantly)."
```

This comes from the browser compat section of the docs, now removed:

> When changing the gain value of a {{domxref("GainNode")}}, Google Chrome prior to version 64 (January 2018) would perform a smooth interpolation to prevent dezippering.
Starting with version 64, the value is changed instantly to bring it in line with the Web Audio spec.
See [Chrome Platform Status](https://chromestatus.com/feature/5287995770929152) for details.

---

According to the [Chrome Platform Status](https://chromestatus.com/feature/5287995770929152), dezippering was removed in Chrome 66.

I found https://github.com/chromium/chromium/commit/b9b924b4656a95b54f2a1182cb5386b6b7d089c8#diff-10017f12d120b088446f883b67696f7dcb96444a7f0ccc44d0bb7c90c7ffd81e, which landed in [66.0.3341.0](https://github.com/chromium/chromium/releases/tag/66.0.3341.0) before 66.0.3359.

I think the 64 version comes from the deprecated warning, which [mentioned removal in Chrome 64](https://github.com/chromium/chromium/commit/ecb779ec2d7c646046f9c560ec064fc254e8f3e5) first, and later [removal in Chrome 65](https://github.com/chromium/chromium/commit/25c2cecdfa4f68a1e83c65c04bcff97152b00cd9#diff-3c30bbba11f9f00e3e37df6ded6f3ab51fddad731e3891345418d9f1271db370L182), but finally it was removed only in Chrome 66 (see commit above).



```suggestion
              "notes": "Before Chrome 66, the gain value of a `GainNode` would perform a smooth interpolation to prevent dezippering (instead of changing instantly)."
```

---

@caugner I'm happy with this. Note though, in the past I have been told that I should use "version" not "chrome" since the user agent is known by the parent of this item. So I'll change that. If you think I'm wrong, then please revert and merge :-)

```suggestion
              "notes": "Before version 66, the gain value of a `GainNode` would perform a smooth interpolation to prevent dezippering (instead of changing instantly)."
```

---

I guess the "version" not "chrome" rule came from a limitation where this note was mirrored as is to downstream browsers, but @queengooborg recently fixed this in https://github.com/mdn/browser-compat-data/pull/24910, so "Before Chrome 66" is mirrored to "Before Edge x" where x is the Edge version corresponding to Chrome 66.

---

It's true that the browser can be assumed as context, but I think it's slightly more readable with "Chrome" and reduces the likelihood of accidentally adding a note for the wrong browser.

---

Actually, `version` and `[browser name]` were both applicable and established long before mirroring was a thing!

---

@caugner I don't mind either way - just let me know the rule and I'll try stick to it. At the moment I'll use "version" until told otherwise.